src/guide/components/props.md

```diff
@@ -593,7 +593,7 @@ When a prop is declared to allow multiple types, e.g.
 
 ```js
 defineProps({
-  disabled: [Boolean, Number]
+  disabled: [String, Boolean, Number]
```

Comment:
I'm not sure why these examples need to change. They seem fine to me as they were.